conf/helmfile.d/0310.tf-serving.yaml

```diff
@@ -88,7 +87,7 @@ releases:
         MODEL_CONFIG_FILE: /kiosk/tf-serving/models.conf
         BATCHING_CONFIG_FILE: /kiosk/tf-serving/batching_config.txt
         ENABLE_BATCHING: "true"
-        MAX_BATCH_SIZE: 1
+        MAX_BATCH_SIZE: 128
```

Comment:
Do we do any kind of stress testing of the tf-serving pipeline in the unit or integration tests? If not, is that worth having an open issue for?

Reply:
We can add it as a TODO in the tests issue we currently have (#330). 

I think automated stress testing could become expensive, though.

Reply:
I think we set this as a maximum value for 128x128 image patches (the smallest size of our current models) based on the available memory and the size of a (128,128,1) float32 image.

Reply:
That makes sense.